apps/ledger-live-desktop/src/renderer/components/Onboarding/Screens/Welcome/index.tsx

```diff
@@ -153,7 +169,7 @@ export function Welcome() {
           </Description>
         </Presentation>
         <ProductHighlight>
-          {isFeatureFlagsSettingsButtonDisplayed && (
```

Comment:
"!" will be removed in another PR link to this one

Reply:
Can you already remove it please ? Just to make sure that we won't forget it